addons/account/models/product.py

```diff
@@ -10,15 +10,23 @@
 class ProductCategory(models.Model):
     _inherit = "product.category"
 
+    def _default_property_account_income_categ_id(self):
+        self.env.company.account_income_id
+
+    def _default_property_account_expense_categ_id(self):
+        self.env.company.account_expense_id
```

Comment:
Unless python changed a lot recently, those methods do not return anything :thinking:

Reply:
oops 🤦‍♂️